apps/web/src/shared/modal/types/burn/ModalBurn.styled.js

```diff
@@ -0,0 +1,170 @@
+import styled from 'styled-components';
+import { m } from 'framer-motion';
+
+export const _container = styled(m.div)``;
+export const _root_container = styled(_container)`
+  backdrop-filter: ${({ filter }) => filter};
+  transform: scale(0.95);
+  -webkit-font-smoothing: subpixel-antialiased;
+  margin-top: 56px;
+  backface-visibility: hidden;
+`;
+export const _container_center = styled(_container)`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: ${({ gap }) => gap};
+  margin-top: ${({ marginTop }) => marginTop};
+`;
+export const _nft_container = styled(_container_center)`
+  > div > div {
+    margin: 0;
+  }
+  transform: scale(0.9);
+  backface-visibility: hidden;
+`;
+
+export const _resource_gain = styled(_container_center)`
+  margin-top: 8px;
+  margin-bottom: 8px;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  gap: 1em;
+`;
+export const _resource = styled(_container_center)`
+  flex-direction: row;
+  gap: 0.2em;
+  > span {
+    color: ${({ theme }) => theme.text['base']};
+  }
+  > svg {
+    width: 32px;
+    height: 32px;
+  }
+`;
+export const _button = styled(m.div)`
+  width: 90px;
+  aspect-ratio: 2/1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 16px;
+  color: ${({ theme }) => theme.text['base']};
+  background: ${({ theme }) => theme.background['base']};
+  box-shadow: ${({ theme }) => theme.shadow['glass']};
+  cursor: pointer;
+  font-size: x-large;
+  transition: all 0.1s ease-in-out;
```

Comment:
We'll have to refactor all css animations to framer-motion at some point for performance, will show you how

Reply:
will do, ill also start some tertiary research